docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  canarie-api:
+    build: .
+    ports:
+      - '2000:2000'
+
+  canarie-api-cron:
+    build: .
+    command: cron -f
+    environment:
+      # set CANARIE_PARSE_LOGS to parse proxy logs (proxy must be running and sharing logs as a volume)
+      - CANARIE_PARSE_LOGS=1
+      # set CANARIE_MONITOR to monitor whether a registered service is up or not
+      - CANARIE_MONITOR=1
+    depends_on:
+      - canarie-api
+    volumes:
+      - access_log:/opt/local/src/access_file.log
+
+  proxy:
+    image: nginx
+    volumes:
+      - access_log:/var/log/nginx/access.log
+
+volumes:
+  access_log:
```

Comment:
Considering log rotation that could contain multiple files, it would probably be better to mount a dedicated directory.

I think it would be wise also to provide an example (i.e.: what is expected) of a configured log rotation of nginx here using the docker service definition for reference (e.g.: https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/config/proxy/docker-compose-extra.yml#L3-L8, but **not** using `driver: json-file` so the parsing still works in CanarieAPI). 

I think the log rotation aspect could be improved (not necessarily in this PR, but setup for it).
Log parsers should probably count everything in the "current log" and add them to call counts from older logs, which are gradually wiped. 
When the cron job is called, only log entries after the latest update would be parsed to avoid re-counting the same entries twice.

Reply:
Thanks, that all makes a lot of sense! But... from our previous discussion I got the sense that the log monitoring isn't used much, if at all. Do you think that it is worth the effort to make the log rotation work really well if there is a chance we might drop support for that feature in the near future? 

If you think there is a use-case for it still I'll leave it in and make the changes you suggest. Let me know what you think.

Reply:
Specifically for the case of nginx requests, there are a lot of logs. There must be some form of log rotation, or the proxy will very quickly run into full disk error. This is the reason the referenced logging definition was originally added.

I don't mind not putting too much effort on parsing the log rotation, but I think it could be added easily. 
The issue I can see with the PR is that since CanarieAPI is no longer in charge of managing the rotation, logs that will be parsed when the cron is called will not be in sync with whichever rotation is applied on the proxy docker. But like you said, those stats are not used that much, so maybe not that much of an issue.